### PR TITLE
Fix the Chrono Miner warp sequences.

### PR DESCRIPTION
**Pull request by GraionDilach**
This pretty much made the two animations playing on top of each other.

### CODE REVIEW

**Comment by GraionDilach** [comment]
Updated with a commit to fix the warp sequences - in RA2, the warp animation played where the Chrono Miner jumped from and not in front of the refinery.

File: rules/allied-vehicles.yaml [hunk 1]
```diff
@@ -61,8 +61,8 @@ cmin:
 		HarvestVoice: Move
 		DeliverVoice: Move
 	ChronoResourceDelivery:
-		WarpInSequence: # unused
-		WarpOutSequence: warpout
+		WarpInSequence: warpout
+		WarpOutSequence: # unused
```
Comment (abcdefg30) [inline rules/allied-vehicles.yaml]: Just remove this entirely then, please.

Comment (GraionDilach) [inline rules/allied-vehicles.yaml]: Done.

**Comment by abcdefg30** [comment]
Looks good to me, thanks. :+1: / :white_check_mark:

File: rules/allied-vehicles.yaml [hunk 2]
```diff
@@ -61,8 +61,7 @@ cmin:
 		HarvestVoice: Move
 		DeliverVoice: Move
 	ChronoResourceDelivery:
-		WarpInSequence: # unused
```
Comment (Iamgoofball) [inline rules/allied-vehicles.yaml]: I'm pretty sure the chrono miner had separate warp in/warp out visuals. Maybe I'm wrong, I'll check.

Comment (GraionDilach) [inline rules/allied-vehicles.yaml]: You're wrong, the warpin.shp was unused and the game didn't played a warpanim in the refinery. The example gif in https://github.com/OpenRA/ra2/issues/134 showcases this quite well.

Comment (Iamgoofball) [inline rules/allied-vehicles.yaml]: ¯\_(ツ)_/¯

**Pull request merged**